EDF.py

```diff
@@ -308,74 +427,84 @@ def readBlock(self, block):
             assert(fid.tell() == 0)
             blocksize = np.sum(chan_info['n_samps']) * meas_info['data_size']
             fid.seek(meas_info['data_offset'] + block * blocksize)
+            
             for i in range(meas_info['nchan']):
                 buf = fid.read(chan_info['n_samps'][i]*meas_info['data_size'])
-                raw = np.asarray(unpack('<{}h'.format(chan_info['n_samps'][i]), buf), dtype=np.float32)
+                
+                raw = np.asarray(
+                    unpack('<{}h'.format(
+                        chan_info['n_samps'][i]), buf), dtype=np.float32)
+                
                 raw *= self.calibrate[i]
                 raw += self.offset[i]  # FIXME I am not sure about the order of calibrate and offset
                 data.append(raw)
         return data
 
-    def readSamples(self, channel, begsample, endsample):
+    def read_samples(self, channel, begsample, endsample):
         meas_info = self.meas_info
         chan_info = self.chan_info
         n_samps = chan_info['n_samps'][channel]
         begblock = int(floor((begsample) / n_samps))
         endblock = int(floor((endsample) / n_samps))
-        data = self.readBlock(begblock)[channel]
+        data = self.read_block(begblock)[channel]
         for block in range(begblock+1, endblock+1):
-            data = np.append(data, self.readBlock(block)[channel])
+            data = np.append(data, self.read_block(block)[channel])
         begsample -= begblock*n_samps
         endsample -= begblock*n_samps
-        return data[begsample:(endsample+1)]
+        return data[begsample:(endsample + 1)]
 
-####################################################################################################
-# the following are a number  of helper functions to make the behaviour of this EDFReader
-# class more similar to https://bitbucket.org/cleemesser/python-edf/
-####################################################################################################
```

Comment:
don't delete this

Reply:
This section is covered in details in the added documentation -
`        Helper Methods: The following are a number of helper functions to make 
                        the behaviour of this EDFReader class more similar to 
                        https://bitbucket.org/cleemesser/python-edf/
            get_signal_text_labels: Convert Signal Text Labels from unicode 
                                    to strings.
            get_n_signals:      Get Number of Channels.
            get_signal_freqs:   Get Signal Frequencies.
            get_n_samples:      Get Total Number of Samples.
            read_signal:        Reads Entire Signal Record and returns the 
                                values as an array.
`

Reply:
There is value to having the documentation close to the relevant code.